proto/src/main/proto/org/hyades/repometaanalysis/v1/repo_meta_analysis.proto

```diff
@@ -27,11 +27,37 @@ message AnalysisResult {
   optional google.protobuf.Timestamp published = 4;
 }
 
+message IntegrityResult {
+  // The Component for which the integrity check was done
+  Component component = 1;
+  //md5 hash match status of component
+  HashMatchStatus md5_hash_match = 2;
+  //sha1 hash match status of component
+  HashMatchStatus sha1_hash_match = 3;
+  //sha256 hash match status of component
+  HashMatchStatus sha256_hash_match = 4;
+  //Identifier of the repository with which the hashes were compared
+  string repository = 5;
```

Comment:
The information currently being sent here is the repository URL (e.g. `https://repo1.maven.org/maven2/`). Is this intended? If so, both the field name and description should reflect that.